qhub/template/{{ cookiecutter.repo_directory }}/templates/jupyterhub_config.py.j2

```diff
@@ -88,7 +88,7 @@ def qhub_configure_profile(username, safe_username, profile):
                   "chown {uid}:{primary_gid} /mnt/home/{username}",
                   "mkdir -p /mnt/home/shared",
                   "chmod 777 /mnt/home/shared"
```

Comment:
```suggestion
                  "chmod 2777 /mnt/home/shared"
```
Actually I think that I should have changed this line instead...

Reply:
Nope you got the previous one completely right! I've been working on testing this PR.

Reply:
yeah, indeed changing that line will not grant the correct groups permissions for each folder (instead I think root will receive it right?). Nvm, thanks for the reply